difftest/difftest.cc

```diff
@@ -153,7 +153,8 @@ void DifftestRef::set_regs(diff_context_t *ctx, bool on_demand) {
     state->satp->write(ctx->satp);
   }
   if (!on_demand || state->mip->read() != ctx->mip) {
-    state->mip->write(ctx->mip);
+    //state->mip->write(ctx->mip);
+    //printf("not cp mip\n");
```

Comment:
Please also delete this unused printf and put comments in the code, explaining why we should not copy the mip register

Reply:
I checked the NEMU code. NEMU now does not use raise_interrupt API but difftest_skip_ref when clinet is triggered,
so should I fix nemu,but copy mip there may be a risk of letting the spike additionally trigger an interrupt